From: Dave Anderson <anderson@redhat.com>
Date: Tue, 2 Sep 2008 13:49:54 -0400
Subject: [xen] ia64 oprofile: recognize Montvale cpu as Itanium2
Message-id: 48BD7CC2.4060301@redhat.com
O-Subject: [RHEL5.3 PATCH 2/2 v2] oprofile: recognize ia64 Montvale cpu as Itanium2 (xen)
Bugzilla: 452588

xen version.

diff --git a/arch/ia64/xen/oprofile/perfmon.c b/arch/ia64/xen/oprofile/perfmon.c
index f9298e0..6880a4d 100644
--- a/arch/ia64/xen/oprofile/perfmon.c
+++ b/arch/ia64/xen/oprofile/perfmon.c
@@ -85,6 +85,7 @@ static char * get_cpu_type(void)
         case 0x07:
             return "ia64/itanium";
         case 0x1f:
+        case 0x20:
             return "ia64/itanium2";
         default:
             return "ia64/ia64";
